packages/gatsby-core-utils/src/fetch-remote-file.ts

```diff
@@ -72,9 +74,63 @@ const ERROR_CODES_TO_RETRY = [
   `ERR_GOT_REQUEST_ERROR`,
 ]
 
+/********************
+ * Queue Management *
+ ********************/
+
+const GATSBY_CONCURRENT_DOWNLOAD = process.env.GATSBY_CONCURRENT_DOWNLOAD
+  ? parseInt(process.env.GATSBY_CONCURRENT_DOWNLOAD, 10) || 0
+  : 50
+
+const q: queue<IFetchRemoteFileOptions> = Queue(
+  fetchWorker,
+  GATSBY_CONCURRENT_DOWNLOAD
+)
+
+/**
+ * fetchWorker
+ * --
+ * Handle fetch requests that are pushed in to the Queue
+ */
+async function fetchWorker(
+  task: IFetchRemoteFileOptions,
+  cb: done
+): Promise<unknown> {
+  try {
+    const node = await fetchFile(task)
+    return cb(null, node)
+  } catch (e) {
+    return cb(e)
+  }
+}
```

Comment:
The type here should be void and done needs to be a string
```suggestion
async function fetchWorker(
  task: IFetchRemoteFileOptions,
  cb: done<string>
): Promise<void> {
  try {
    const node = await fetchFile(task)
    return void cb(null, node)
  } catch (e) {
    return void cb(e)
  }
}
```

Reply:
adjusted!